app/form_models/publishers/job_listing/application_form_form.rb

```diff
@@ -2,54 +2,19 @@ class Publishers::JobListing::ApplicationFormForm < Publishers::JobListing::Vaca
   validate :application_form_presence
   validates :application_form, form_file: DOCUMENT_VALIDATION_OPTIONS
 
-  validates :application_email, presence: true
-  validate :other_application_email_presence
-  validate :other_application_email_valid
-
-  def self.fields
-    %i[application_email]
-  end
-  attr_accessor(:application_form, :application_form_staged_for_replacement, *fields)
-  attr_writer(:other_application_email)
-
-  def application_email
-    return unless @vacancy.application_email || params[:application_email]
-
-    if params[:application_email].present?
-      return params[:application_email] if params[:application_email] == @current_publisher&.email
-
-      return "other"
-    end
-
-    return @current_publisher&.email if @vacancy.application_email == @current_publisher&.email
-
-    "other"
-  end
-
-  def other_application_email
-    return params[:other_application_email] if params[:other_application_email]
-
-    @vacancy.application_email unless @vacancy.application_email == @current_publisher&.email
-  end
+  attr_accessor(:application_form, :application_form_staged_for_replacement)
```

Comment:
minor, but its more idiomatic to use attributes that attr_accessors. Doesn't make a lot of difference here TBH